### PR TITLE
Add build actions

### PR DESCRIPTION
**Pull request by ctan-g**
This commit adds GitHub CI build actions.

### CODE REVIEW

**Comment by ctan-g** [comment]
P.S.: Current action results are here: https://github.com/ctan-g/silk/actions/runs/203419286

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+        if: "github.ref != 'refs/heads/master'"
+
+  build:
+    name: Build (Python ${{ matrix.python-version }}, ${{ matrix.os }})
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8]
```
Comment (simonlingoogle) [inline .github/workflows/build.yml]: Are we not planning to support Python 3.6?

Comment (ctan-g) [reply .github/workflows/build.yml]: Good point, will add.

**Review by simonlingoogle — APPROVED** [review]
LGTM :+1:

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+        if: "github.ref != 'refs/heads/master'"
+
+  build:
+    name: Build (Python ${{ matrix.python-version }}, ${{ matrix.os }})
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+        os: [ubuntu-latest, macos-latest]
```
Comment (qilinkang) [inline .github/workflows/build.yml:34]: Can we add debian os ?

Comment (simonlingoogle) [reply .github/workflows/build.yml:34]: https://docs.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners#supported-runners-and-hardware-resources

I believe there is no debian OS support.

**Review by qilinkang — APPROVED** [review]
LGTM

**Review by jwhui — APPROVED** [review]

**Pull request merged**